pkg/deps/config/suppliers.go

```diff
@@ -466,12 +479,31 @@ func NewSupplyBankQuerierFn() SupplierFn {
 
 // newSupplyTxClientFn returns a new depinject.Config which is supplied with
 // the given deps and the new TxClient.
-func newSupplyTxClientsFn(ctx context.Context, deps depinject.Config, signingKeyName string) (depinject.Config, error) {
+func newSupplyTxClientsFn(
+	ctx context.Context,
+	deps depinject.Config,
+	signingKeyName string,
+	gasPrices cosmostypes.DecCoins,
+) (depinject.Config, error) {
+	// Ensure that the gas prices include upokt
+	uPOKTDenomFound := false
+	for _, gasPrice := range gasPrices {
+		if gasPrice.Denom == volatile.DenomuPOKT {
+			uPOKTDenomFound = true
+			break
+		}
```

Comment:
1. If it's something else, can we log it?
2. WHen would something else show up here?